api_tests/base/test_filters.py

```diff
@@ -467,6 +474,56 @@ def test_filter_queryset_handles_multiple_fields(self):
             )]
         assert_equal(actual, [40, 30, 10, 20])
 
+    def test_sort_by_serializer_field(self):
+        user = AuthUserFactory()
+        node_one = NodeFactory(creator=user)
+        node_two = NodeFactory(creator=user)
+
+        # Ensuring that sorting by the serializer field returns the same result
+        # as using the source field
+        res_created = self.app.get('/{}nodes/?sort=created'.format(API_BASE), auth=user.auth)
+        res_date_created = self.app.get('/{}nodes/?sort=date_created'.format(API_BASE), auth=user.auth)
+        assert res_created.status_code == 200
+        assert res_created.json['data'] == res_date_created.json['data']
+        assert res_created.json['data'][0]['id'] == str(node_one._id)
+        assert res_date_created.json['data'][0]['id'] == node_one._id
+        assert res_created.json['data'][1]['id'] == node_two._id
+        assert res_date_created.json['data'][1]['id'] == node_two._id
+
+        # Testing both are capable of using the inverse sort sign '-'
```

Comment:
Nitpicky but I think this should simply be another test method given that it's testing something distinct